### DISCUSSION TITLE
Optimized Trend Tracker

### DISCUSSION THREAD

**Discussion by wclark17** [DaveSkender/Stock.Indicators#677] [Ideas]
Hi - 
  Not sure how much time, or interest in this, you may have, but I have been working with new(?) indicator that is referred to as the Optimized Trend Tracker.  I think there are a couple of different implementations, but there is python code here, https://github.com/ysdede/jesse_strategies/tree/master/OttBands5minFixedOtt-2-MATIC/custom_indicators, that implements one version.  
  Any chance you can code this into your library?

Thanks
Bill

**Comment by wclark17** [comment]
I found it on Tradingview.com as well, and many derivatives, but this looks like the first:
https://www.tradingview.com/script/zVhoDQME/

Interested in your thoughts.

**Reply by DaveSkender** [reply]
I'll take a look.  I generally don't implement community-created indicators from TradingView or similar places simple because they exist, since they're mostly just random inventions of novices.  Do you have any descriptive references or links from more reputable sources, like trade magazines, books, or known professionals/authors?

**Comment by wclark17** [comment]
Doing a Google search for 'Optimized Trend Tracker Indicator' shows
discussion and use and a couple other websites like mql5, but it seems
overall pretty new.and a couple of YouTube videos.  Sadly nothing of sites
you mentioned though.

Appreciate it if you have time to give it a quick look anyway.

Thanks!

**Comment by wclark17** [comment]
I actually ended up writing this (I think, still need to test some)
And it turns out, it needed 2 other indicators as well.
1) Variable Index Dynamic Range: https://www.metatrader5.com/en/terminal/help/indicators/trend_indicators/vida  - This seems like a custom indicator
2) Change Momentum Oscillator: https://www.investopedia.com/terms/c/chandemomentumoscillator.asp - This one has been around since 1994 and seems to be more established.

Happy to share the code if you are interested.  Also understand if you want to pass on these.

-Bill

**Reply by DaveSkender** [reply]
Yah, anything from Chande is good.  He's a reputable author in this space.  Go ahead and add an Issue for that one.  I'll look at the other one too.

**Reply by DaveSkender** [reply]
CMO was implemented in #893 for version 2.3

**Reply by wclark17** [reply]
Great!  Thanks!
